Source/ACE/Network/GameEvent/Events/GameEventIdentifyObjectResponse.cs

```diff
@@ -1,15 +1,24 @@
-﻿namespace ACE.Network.GameEvent.Events
+﻿using ACE.Network.Enum;
+
+namespace ACE.Network.GameEvent.Events
 {
     public class GameEventIdentifyObjectResponse : GameEventMessage
     {
         public GameEventIdentifyObjectResponse(Session session, uint objectID)
             : base(GameEventType.IdentifyObjectResponse, GameMessageGroup.Group09, session)
         {
-            // TODO
-
+            // TODO - Send Actual Properties
             Writer.Write(objectID);
-            Writer.Write(0u);
-            Writer.Write(1u);
+
+            IdentifyResponseFlags flags = IdentifyResponseFlags.None;
+            flags = IdentifyResponseFlags.StringStatsTable;
+            Writer.Write((uint)flags); // Flags
+            Writer.Write(1u); // Success bool
+
+            // Write a simple debug strings. Thanks to Pea for this idea.
+            Writer.Write(1u + (1 << 16)); // One String, + table size
+            Writer.Write(16u); // Long Description String
+            Writer.WriteString16L("This item has a guid of " + objectID.ToString() + " (0x" + objectID.ToString("X") + ")");
         }
     }
 }
```

Comment:
Is there a way to add this to the StyleCop settings? It didn't throw up any alert on this. I have made a mental note, however, for the future.